Agreements/Airport/23_GD.txt

```diff
@@ -1,3 +1,3 @@
-COPX:*:*:ABDAL:EGFF:*:London S23:GDUP:*:12000:|ABDAL
-COPX:*:*:ABDAL:EGSY:*:London S23:GDUP:*:12000:|ABDAL
-COPX:*:*:POMAX:EGGD:*:London S23:GDUP:*:12000:|POMAX
+COPX:BRI:*:*:EGFF:*:London S23:GDUP:*:12000:|ABDAL
+COPX:BRI:*:*:EGSY:*:London S23:GDUP:*:12000:|ABDAL
+COPX:POMAX:*:*:EGGD:*:London S23:GDUP:*:12000:|POMAX
```

Comment:
Bit confused about the top two - do these display prior to BRI or only after?

Reply:
Going to leave the Bristol one as it was, seems it was just a spot error